frontend/src/container/FormAlertRules/QuerySection.tsx

```diff
@@ -27,7 +27,7 @@ function QuerySection({
 
 	const handleQueryCategoryChange = (s: string): void => {
```

Comment:
```suggestion
	const handleQueryCategoryChange = (query: string): void => {
```


i know not it's not your change but good to have